--- thunar/Makefile.am	2021-11-22 15:30:07.000000000 -0500
+++ mod/thunar/Makefile.am	2021-12-03 14:52:35.089460050 -0500
@@ -281,10 +281,10 @@
 	$(AM_V_GEN) gdbus-codegen --c-namespace Thunar --generate-c-code=thunar-dbus-service-infos $(srcdir)/thunar-dbus-service-infos.xml
 
 thunar-thumbnailer-proxy.h thunar-thumbnailer-proxy.c: $(srcdir)/thunar-thumbnailer-dbus.xml Makefile
-	$(AM_V_GEN) gdbus-codegen --c-namespace Thunar --generate-c-code thunar-thumbnailer-proxy $(srcdir)/thunar-thumbnailer-dbus.xml
+	$(AM_V_GEN) gdbus-codegen --c-namespace Thunar --generate-c-code=thunar-thumbnailer-proxy $(srcdir)/thunar-thumbnailer-dbus.xml
 
 thunar-thumbnail-cache-proxy.h thunar-thumbnail-cache-proxy.c: $(srcdir)/thunar-thumbnail-cache-dbus.xml Makefile
-	$(AM_V_GEN) gdbus-codegen --c-namespace Thunar --generate-c-code thunar-thumbnail-cache-proxy $(srcdir)/thunar-thumbnail-cache-dbus.xml
+	$(AM_V_GEN) gdbus-codegen --c-namespace Thunar --generate-c-code=thunar-thumbnail-cache-proxy $(srcdir)/thunar-thumbnail-cache-dbus.xml
 
 thunar-marshal.h: stamp-thunar-marshal.h
 	@true
