diff -uNr zenity-ZENITY_3_32_0/src/main.c downloads/zenity-ZENITY_3_32_0/src/main.c
--- ./src/main.c     2019-03-22 15:37:18.000000000 +0530
+++ ./src/main.c   2021-05-10 16:48:20.073985604 +0530
@@ -44,7 +44,9 @@
 #endif

        bindtextdomain (GETTEXT_PACKAGE, GNOMELOCALEDIR);
+#ifndef __ANDROID__
        bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
+#endif
        textdomain (GETTEXT_PACKAGE);
                                                                   gtk_init (&argc, &argv);
