*** st-0.8.2/config.mk	2019-07-09 08:09:36.007814125 -0700
--- st-0.8.2/config.mk	2019-07-09 08:09:52.917814118 -0700
***************
*** 16,22 ****
  INCS = -I$(X11INC) \
         `$(PKG_CONFIG) --cflags fontconfig` \
         `$(PKG_CONFIG) --cflags freetype2`
! LIBS = -L$(X11LIB) -lm -lrt -lX11 -lutil -lXft \
         `$(PKG_CONFIG) --libs fontconfig` \
         `$(PKG_CONFIG) --libs freetype2`
  
--- 16,22 ----
  INCS = -I$(X11INC) \
         `$(PKG_CONFIG) --cflags fontconfig` \
         `$(PKG_CONFIG) --cflags freetype2`
! LIBS = -L$(X11LIB) -lm -lrt -lX11 -lXft \
         `$(PKG_CONFIG) --libs fontconfig` \
         `$(PKG_CONFIG) --libs freetype2`
  
