diff -uNr SDL2-GNUBoy/makefile SDL2-GNUBoy.mod/makefile
--- SDL2-GNUBoy/makefile        2021-11-13 23:17:46.675435490 -0600
+++ SDL2-GNUBoy.mod/makefile    2021-11-13 23:20:52.567435357 -0600
@@ -1,4 +1,4 @@
-.PHONY: all
+.PHONY: linux
 .PHONY: clean

 .PHONY: linux
@@ -22,4 +22,4 @@
        rm -f sys/*/*.o src/*.o lib/*/*.o

 clean:
-       rm -f *gnuboy sdl2gnuboy.exe mac-sdl2gnuboy gmon.out *.o sys/*.o lib/*/*.o sys/*/*.o src/*.o
\ No newline at end of file
+       rm -f *gnuboy sdl2gnuboy.exe mac-sdl2gnuboy gmon.out *.o sys/*.o lib/*/*.o sys/*/*.o src/*.o
