diff -uNr i3-4.20.1/i3bar/src/main.c i3-4.20.1.mod/i3bar/src/main.c
--- i3-4.20.1/i3bar/src/main.c
+++ i3-4.20.1.mod/i3bar/src/main.c
@@ -160,7 +160,7 @@
     }
 
     if (socket_path == NULL) {
-        char *i3_default_sock_path = "/tmp/i3-ipc.sock";
+        char *i3_default_sock_path = "@TERMUX_PREFIX@/tmp/i3-ipc.sock";
         ELOG("No socket path specified, default to %s\n", i3_default_sock_path);
         socket_path = sstrdup(i3_default_sock_path);
     }
